### DISCUSSION TITLE
Scrollbar for long images?

### DISCUSSION THREAD

**Discussion by deanloew** [viliusle/miniPaint#376] [General]
Is there a way to enable a scrollbar for images with great height?  I am working on some images that are 4000 or more pixels high, but I do not see how to easily move back and forth from the top to the bottom of the image.    I can zoom in and out, and sort of direct the zoom to which way I want to zoom.

Does the miniPaint canvas support either horizontal or vertical scrollbars?

Thank you!
Dean

**Comment by viliusle** [comment]
We don't have scroll bars, only because we have preview window (on top right corner), where you can click to move visible area position where you clicked, also you can drag. Try it.
![11](https://github.com/viliusle/miniPaint/assets/3708786/45a74e07-0f6e-4956-b1f4-0ab6ac4379c9)

**Comment by deanloew** [comment]
Excellent!  I did not even know about that.  Better than a pair of
scrollbars.   Thank you!
